doc/tools.rst

```diff
@@ -134,6 +134,20 @@ not complicated::
         The number of blocks in active use that have been allocated
         through this pool.
 
+    .. attribute:: managed_bytes
```

Comment:
Imho, it's a bit confusing that it's `held_blocks`, but not `held_bytes`, but doesn't matter much I think.

Reply:
I agree. However `held_bytes` is harder to track (/needs to be updated in more places) than these two, so I chose managed/active as the internal state. And computing `held_bytes` as `managed - active` is not correct, since active allocations are counted with their application-requested size, not their allocation size.